package.json

```diff
@@ -14,7 +14,11 @@
     "preview": "rsbuild preview"
   },
   "browserslist": {
-    "production": [">0.2%", "not dead", "not op_mini all"],
```

Comment:
@misicnenad I think you're missing some formater related changes? I recall this bit was just formatted to the original code here and this PR reverts it back.

Reply:
I definitely am missing something. I left a related comment on one of the recent PRs https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2707/files#r1481187581

@ap-justin do you know why this might be happening?

Reply:
@misicnenad, @SovereignAndrey - might be because we are using biome, but our vscode extension is still prettier. I am using the biome vscode extension and `biome check` and format on save is the same
<img width="733" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/1b638105-78a1-4347-b66d-81b878740c47">

Reply:
I removed `prettier` extension and installed `biome` as soon as we made the switch

Reply:
Hey now! Don't lump all me in with that VS code stuff. ;)

Reply:
Very retro of you Andrey :P

Reply:
@ap-justin I figured out *when* this formatting happens - whenever I run `yarn install`, do you know what's causing this to happen? :open_mouth: Opening *package.json* and formatting the file inlines this back :thinking: